lib/teracy-dev-essential/config/rsync_recovery.rb

```diff
@@ -115,6 +115,13 @@ def valid_command(plugins)
       def configure_trigger_after(config)
         config.trigger.after :up, :reload, :resume do |trigger|
           trigger.ruby do |env, machine|
+            ready = machine.guest.ready?
+
+            unless ready
+              @logger.warn("guest machine is not ready currently")
```

Comment:
@logger.error("guest machine is not ready")

abort when error, otherwise, don't abort.